ideascube/mediacenter/management/commands/import_medias.py

```diff
@@ -93,21 +93,32 @@ def add(self, metadata):
                                 title)
             return
 
-        path = os.path.join(self.ROOT, original)
-        if not os.path.isfile(path):
-            self.report.error(u'Path not found', path)
-            return
-        with open(path, 'rb') as f:
-            original = File(f, name=original)
-
+        original_path = os.path.join(self.ROOT, original)
+        try:
+            with open(original_path, 'rb') as f:
+                original = File(f, name=original)
+
+                preview = metadata.get('preview')
+                if preview:
+                    preview_path = os.path.join(self.ROOT, preview)
+                    try:
+                        with open(preview_path, 'rb') as f:
+                            preview = File(f, name=preview)
+                            self.save(metadata, original, instance, preview)
+                    except IOError:
+                        self.report.error("Cannot open path", preview_path)
+                else:
+                    self.save(metadata, original, instance)
+        except IOError:
+            self.report.error("Cannot open path", original_path)
             preview = metadata.get('preview')
             if preview:
-                path = os.path.join(self.ROOT, preview)
-                with open(path, 'rb') as f:
-                    preview = File(f, name=preview)
-                    self.save(metadata, original, instance, preview)
-            else:
-                self.save(metadata, original, instance)
+                preview_path = os.path.join(self.ROOT, preview)
+                try:
+                    with open(preview_path, 'rb') as f:
+                        pass
+                except IOError:
+                    self.report.error("Cannot open path", preview_path)
```

Comment:
The main difficulty here is that we need to have both files opened when we are calling the `save()` method.
So we need to have the enclosed `with open(...)`.

Or, we can have a simple `open` and close it in a `finaly` close.